Testing/ORKTest/ORKTest/TaskFactory.m

```diff
@@ -0,0 +1,3622 @@
+//
+//  TaskFactory.m
+//  ORKTest
+//
+//  Created by Ricardo Sanchez-Saez on 1/9/17.
+//  Copyright © 2017 ResearchKit. All rights reserved.
+//
+
+#import "TaskFactory.h"
+
+#import "ORKTest-Swift.h"
+
+#import "DragonPokerStep.h"
+#import "DynamicTask.h"
+
+@import ResearchKit;
+
+
+/**
+ A subclass is required for the login step.
+ 
+ The implementation below demonstrates how to subclass and override button actions.
+ */
+@interface LoginViewController : ORKLoginStepViewController
+
+@end
+
+@implementation LoginViewController
+
+- (void)forgotPasswordButtonTapped {
+    UIAlertController *alert = [UIAlertController alertControllerWithTitle:@"Forgot password?"
+                                                                   message:@"Button tapped"
+                                                            preferredStyle:UIAlertControllerStyleAlert];
+    [alert addAction:[UIAlertAction actionWithTitle:@"OK" style:UIAlertActionStyleDefault handler:nil]];
+    [self presentViewController:alert animated:YES completion:nil];
+}
+
+@end
+
+
+/**
+ A subclass is required for the verification step.
+ 
+ The implementation below demonstrates how to subclass and override button actions.
+ */
+@interface VerificationViewController : ORKVerificationStepViewController
+
+@end
+
+@implementation VerificationViewController
+
+- (void)resendEmailButtonTapped {
+    UIAlertController *alert = [UIAlertController alertControllerWithTitle:@"Resend Verification Email"
+                                                                   message:@"Button tapped"
+                                                            preferredStyle:UIAlertControllerStyleAlert];
+    [alert addAction:[UIAlertAction actionWithTitle:@"OK" style:UIAlertActionStyleDefault handler:nil]];
+    [self presentViewController:alert animated:YES completion:nil];
+}
+
+@end
+
+
+@implementation TaskFactory
+
+#pragma mark - Mapping identifiers to tasks
+
++ (instancetype)sharedInstance
+{
+    static dispatch_once_t onceToken;
+    static id sharedInstance = nil;
+    dispatch_once(&onceToken, ^{
+        sharedInstance = [[[self class] alloc] init];
+    });
+    return sharedInstance;
+}
+
+#pragma mark - Mapping identifiers to tasks
+
+- (id<ORKTask>)makeTaskWithIdentifier:(NSString *)identifier {
+    if ([identifier isEqualToString:DatePickingTaskIdentifier]) {
```

Comment:
As a part of doing this, could you also alphabetically order the tasks? That would reduce the possibility of merge conflicts (instead of everyone adding to the bottom). Thanks!